package.json

```diff
@@ -23,9 +23,11 @@
 		"url": "https://ancientbeast.com/contribute/#donate"
 	},
 	"dependencies": {
+		"@heroiclabs/nakama-js": "^2.0.1",
```

Comment:
Woah, that's over 2 years old. Will see about poking them to make a new release already...